pendulum_control/src/pendulum_logger.cpp

```diff
@@ -46,13 +46,13 @@ int main(int argc, char * argv[])
       printf("Commanded motor angle: %f\n", msg->command.position);
       printf("Actual motor angle: %f\n", msg->state.position);
 
-      printf("Current latency: %lu ns\n", msg->cur_latency);
+      printf("Current latency: %llu ns\n", (long long) msg->cur_latency);
       printf("Mean latency: %f ns\n", msg->mean_latency);
-      printf("Min latency: %lu ns\n", msg->min_latency);
-      printf("Max latency: %lu ns\n", msg->max_latency);
+      printf("Min latency: %llu ns\n", (long long) msg->min_latency);
```

Comment:
Instead of doing these C-style casts, I'd much prefer if we used the `PRId64` series of macros.  That will ensure it will work on all platforms without casts, and is the solution we've used elsewhere in the codebase.  The same goes for the changes below.

Reply:
Changed in next revision.